### ISSUE TITLE
data modeling

### ISSUE THREAD

**Issue by nsfmc** [studiosnack/brews-clues#3]
Ok, so we have this general idea:
* a brew
* a bag of coffee
* brewing equipment
* a vendor?

what goes into a brew?

**brew**
* **required**
  * date (when you do it, maybe time also?)
  * the coffee you used
* **optional**
  * how much coffee you used
  * the equipment you used
  * how much coffee you made? (or how much water you used)
  * rating: 😄 / ☹️ / 😑 
  * notes
  * the grind setting? (this is super nitpicky, but it does affect things)

**bag of coffee**
* **required**
  * name
  * vendor
  * cost
* **optional**
  * country of origin (could be derived from name) (also not sure how to represent this)
  * notes
  * roast date
  * roast type
  * pr spin from roaster

**equipment**
* **required**
  * name
* **optional**
  * cost
  * date acquired
  * notes

**vendor**
* **required**
  * name
* **optional**
  * foursquare id? (could maybe do some yelp/4sq/swarm integration to pester you to add a bag of coffee after you check in)
  * notes

**Comment by nsfmc** [comment]
how do we handle something like cold brew (where i, at least, just soak a bunch of ground coffee) for ~12 hours?

**Comment by chiaberry** [comment]
would cold brew be under equipment? 

Also, under vendor -- is that for purchase of the beans? Or will this also track coffees purchased at shops?

**Comment by nsfmc** [comment]
yeah, i think we could make coldbrew its own equipment (or a custom diy one since there are both diy and expensive af versions)

no, i don't know if we get much benefit from tracking a pourover i buy somewhere (although maybe?), i was just thinking it was for where you bought your beans so that you could just say "where do my favorite coffees come from?"

**Comment by chiaberry** [comment]
Agree on the vendor. Tracking pourovers and drinks from vendors would be a different type of project.
